build/pkgs/rpy2/spkg-configure.m4

```diff
@@ -1,6 +1,6 @@
 SAGE_SPKG_CONFIGURE([rpy2], [
-    sage_spkg_install_rpy2=yes
-  ], [dnl REQUIRED-CHECK
+    SAGE_SPKG_CONFIGURE([rpy2], [SAGE_PYTHON_PACKAGE_CHECK([rpy2])])
```

Comment:
Nesting SAGE_SPKG_CONFIGURE within SAGE_SPKG_CONFIGURE? That can't be right

Reply:
Ha, no. I was supposed to delete half of that after copy/pasting it in.